lms/templates/discussion/_thread_list_template.html

```diff
@@ -1,17 +1,6 @@
 <%page expression_filter="h"/>
 <%! from django.utils.translation import ugettext as _ %>
 <script type="text/template" id="thread-list-template">
-    <div class="forum-nav-header">
-        <button type="button" class="forum-nav-browse" id="forum-nav-browse" aria-haspopup="true">
```

Comment:
[a11y] [question] does this aria-haspopup="true" item need to be carried over to the All Topics menu item as well?